daisy_workflows/image_build/debian/debian_10_fai.wf.json

```diff
@@ -30,6 +33,42 @@
           "google_cloud_repo": "${google_cloud_repo}"
         }
       }
-    }
+    },
+    "copy-gcs-object": {
+      "CopyGCSObjects": [
+        {
+          "Source": "${OUTSPATH}/root.tar.gz",
+          "Destination": "${OUTSPATH}/root.tar.gz"
+        }
+      ]
+    },
+    "create-image": {
+      "CreateImages": [
+        {
+          "Name": "debian-10-v${build_date}",
+          "RawDisk": {
+            "Source": "${OUTSPATH}/root.tar.gz"
+          },
+          "Description": "Debian, Debian, ${debian_version}, x86_64 built on ${build_date}",
+          "Family": "debian-10",
+          "Project": "${publish_project}",
+          "NoCleanup": true,
+          "ExactName": true
+        }
+      ]
+    },
+    "CreateDisks": [
```

Comment:
i don't think this workflow needs to create a disk, only an image should be OK

Reply:
Why we don't need disk. We use this disk as secondary disk to  create instance for export metadata. The image can only be used to create first boot disk.

Reply:
the github workflows are for customers, and they should result in an image

Reply:
Got it. Resolve